packages/components/package.json

```diff
@@ -36,7 +36,15 @@
   "dependencies": {
     "@akropolis-web/styles": "^0.3.0",
     "@types/classnames": "^2.2.9",
-    "classnames": "^2.0.0"
+    "@types/color": "^3.0.1",
+    "@types/mersenne-twister": "^1.1.2",
+    "@types/react-copy-to-clipboard": "^4.3.0",
+    "classnames": "^2.0.0",
+    "@types/ramda": "^0.27.14",
+    "ramda": "^0.27.1",
+    "mersenne-twister": "^1.1.0",
+    "color": "^3.1.2",
+    "react-copy-to-clipboard": "^5.0.2"
```

Comment:
As I understand, these are peerDependencies for this package and dev dependencies for workspace-root. It's done in this way to be able to develop the package, and use host-project dependencies through peerDependencies, or maybe some of them really unique to the components package, we need to investigate this